### PR TITLE
fix(DB/SAI): updated text for Rune-Inscribed Parchment

### PR DESCRIPTION
**Pull request by Asthalor**
## Changes Proposed:
-  Added the english and french text for the item Rune-Inscribed Parchment.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11583
- Closes https://github.com/chromiecraft/chromiecraft/issues/3440

## SOURCE:

https://classic.wowhead.com/item=9568/rune-inscribed-parchment for the english text
https://fr.classic.wowhead.com/item=9568/parchemin-runique for the french text

## Tests Performed:

- Create lvl 1 shaman
- .levelup 1
- go to gornek, in the cave
- Pick the mission cutting teeth.
- .complete quest 788
- get the rune-inscribed-parchment  quest
- open the letter and check the content

## How to Test the Changes:

1. Create lvl 1 shaman
2. levelup 1
3. go to gornek, in the cave
4.   Pick the mission cutting teeth.
5. .complete quest 788
6. get the rune-inscribed-parchment  quest
7. open the letter and check the content

## Known Issues and TODO List:

- I dont have a french client. Should work tho.

Test image

image

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Maelthyrr** [comment]
Tested with enUS and FR, all work fine.

File: data/sql/updates/pending_db_world/rev_1651751512749712800.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+--
+
+-- Content info for Durotar Shaman Quest [Rune-Inscribed Parchment]
+-- EN/US Text
+UPDATE `page_text`  SET `Text` = "Lok-tar, $gbrother:sister. The elements beckon you closer and bid me to show you the path of the shaman. The spirits of our ancestors watch from beyond and swell with pride knowing you have joined our ranks.$B$BWhen you are ready, seek me out near the entrance to the Den. It is there that I will be training others of our kind. Until then, may the wind be at your back.$B$BShikrik, Shaman Trainer" WHERE id = 2461;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1651751512749712800.sql:5]: Backticks on id

**Review by Kitzunu — CHANGES_REQUESTED** [review]

**Comment by Nefertumm** [comment]
Seems like brother;sister doesn't work. Look at the image you posted

**Comment by Asthalor** [comment]
> Seems like brother;sister doesn't work. Look at the image you posted

Woops, didnt noticed. Im on it

**Comment by Asthalor** [comment]
image

**Review by Nefertumm — APPROVED** [review]

**Pull request merged**